di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/handlers/AuthorizeHandler.java

```diff
@@ -362,6 +370,29 @@ private Map<String, Object> generateGpg45Score(
                     CredentialIssuerConfig.FRAUD_PARAM, Integer.parseInt(fraudValue));
             case VERIFICATION_CRI_TYPE -> gpg45Score.put(
                     CredentialIssuerConfig.VERIFICATION_PARAM, Integer.parseInt(verificationValue));
+            case DOC_CHECK_APP_CRI_TYPE -> {
+                gpg45Score.put(
+                        CredentialIssuerConfig.EVIDENCE_STRENGTH_PARAM,
+                        Integer.parseInt(strengthValue));
+                int validityNum = Integer.parseInt(validityValue);
+                gpg45Score.put(
+                        CredentialIssuerConfig.EVIDENCE_VALIDITY_PARAM,
+                        validityNum);
+
+                List<Map<String, Object>> checkDetailsValue = new ArrayList<>();
+                checkDetailsValue.add(Map.of("checkMethod", "vri"));
+                checkDetailsValue.add(Map.of("checkMethod", "bvr", "biometricVerificationProcessLevel", biometricVerificationValue));
+
+                if (validityNum < 2) {
+                    gpg45Score.put(
+                            CredentialIssuerConfig.FAILED_CHECK_DETAILS_PARAM,
```

Comment:
Does this imply that the app is going to decided if the check was a pass of a fail? Feel like that should be a job for core and its logic.

Reply:
The evidence structure of the gpg45 bit of the VC is slightly different depending on how the check goes so yeah I think the app must do some kind of check.

The evidence block will either have a "checkDetails" property or a "failedCheckDetails" property that contains the biometricVerificationProcessLevel score value.

Reply:
For the stub I've just controlling wether we return "checkDetails"/"failedCheckDetails" on the validity score i.e 2 or 0

Reply:
OK - do you know if this format has been documented in the spec anywhere?

Reply:
I've only seen it here - https://govukverify.atlassian.net/wiki/spaces/DCMAW/pages/3141435393/DCMAW+-+Verifiable+Credential+Response+-+Drivers+License not sure if we have it documented on our VC ADR

Reply:
Actually yeah its on here as well - https://github.com/alphagov/digital-identity-architecture/blob/main/rfc/0024-identity-vc-for-checks.md